typescript/cli/src/tests/warp-deploy.e2e-test.ts

```diff
@@ -0,0 +1,110 @@
+import * as chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+import { ChainAddresses } from '@hyperlane-xyz/registry';
+import { TokenType, WarpRouteDeployConfig } from '@hyperlane-xyz/sdk';
+
+import { WarpSendLogs } from '../send/transfer.js';
+import { writeYamlOrJson } from '../utils/files.js';
+
+import {
+  ANVIL_KEY,
+  REGISTRY_PATH,
+  deploy4626Vault,
+  deployOrUseExistingCore,
+  deployToken,
+  sendWarpRouteMessageRoundTrip,
+} from './commands/helpers.js';
+import { hyperlaneWarpDeploy, readWarpConfig } from './commands/warp.js';
+
+chai.use(chaiAsPromised);
```

Comment:
This is fine but I wonder if we could do this in a mocha test setup file or something to have it apply across all tests instead of just this one

Reply:
https://github.com/chaijs/chai-as-promised/blob/master/test/support/setup.js the solution is doing something like this an importing to each test file. i'll pass on this for now. doesn't seem to be much better than just regular import and then calling `chai.use`